tests/go.mod

```diff
@@ -4,7 +4,32 @@ go 1.20
 
 require (
 	github.com/stretchr/testify v1.8.4
-	github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper v1.10.19
+	github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper v1.11.9
```

Comment:
where are you getting 1.11.9 from? 1.11.6 seems to be the latest

Reply:
Those came automatically from my vscode or whilst running pre-commit hooks